src/data-table/filter-shell.js

```diff
@@ -13,16 +13,33 @@ import {Checkbox, STYLE_TYPE} from '../checkbox/index.js';
 import {useStyletron} from '../styles/index.js';
 import {LocaleContext} from '../locale/index.js';
 
+type ExcludeKind = 'value' | 'range';
+
+// If modifying this, take a look at the histogram and adjust. see HISTOGRAM_SIZE
+const FILTER_SHELL_WIDTH = '320px';
```

Comment:
maybe extract this value out into the `constants.js` file, and subtract some margin when reading in the numerical filter?

Reply:
extracted out to the constants